src/proxy/proxy-handler.js

```diff
@@ -3,10 +3,15 @@ import getStats from 'react-storefront-stats'
 
 export default async function proxyHandler(params, request, response) {
   try {
-    const stats = await getStats()
-    fns.init$(body)
-    renderHeader(stats) // reuse the PWA header in legacy pages
-    response.send($.html())
+		const contentType = env.content_type || '';
+		if (contentType.indexOf('html') > -1) {
+	    const stats = await getStats()
+	    fns.init$(body)
+	    renderHeader(stats) // reuse the PWA header in legacy pages
+	    response.send($.html())
+	  } else {
+	    sendResponse({ htmlparsed: true });
```

Comment:
This affects billing.  We should use htmlparsed: false here when no transformation is done.